app/controllers/CorrectionController.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions.{AuthAction, AuthenticatedControllerComponents}
+import models.requests.CorrectionRequest
+import models.Period
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent}
+import services.CorrectionService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class CorrectionController @Inject()(
+                                      cc: AuthenticatedControllerComponents,
+                                      correctionService: CorrectionService,
+                                      auth: AuthAction
+                                    )(implicit ec: ExecutionContext)
+  extends BackendController(cc) {
+
+  def post(): Action[CorrectionRequest] = cc.authAndCorrectionToggle(parse.json[CorrectionRequest]).async {
+    implicit request =>
+      correctionService.createCorrection(request.body).map {
+        case Some(vatReturn) => Created(Json.toJson(vatReturn))
+        case None => Conflict
+      }
+  }
+
+  def list(): Action[AnyContent] = cc.authAndCorrectionToggle.async {
```

Comment:
Could make this just an overloaded get method on the controller the same as the service methods of the same name. Not fussed either way

Reply:
was keeping it in line with vat return controller (but I may have renamed that)